### PR TITLE
Datagrid: test for fallback image in tooltip

### LINKED ISSUE #6278: Unit Testing: Create a Puppeteer Script for "Datagrid: Add support for Fallback Image Tootip Text"
Describe the Issue
Create a unit testing for "Datagrid: Add support for Fallback Image Tootip Text".

See attached BDD for more information.
[story_github6178.docx](https://github.com/infor-design/enterprise/files/8426434/story_github6178.docx)

Related github issue
#6178

### PR DESCRIPTION
**Pull request by janahintal**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Datagrid add support for fallback image tootip text test

**Related github/jira issue (required)**:

Related to #6178
Closes #6278 

**Steps necessary to review your pull request (required)**:

1. pull this branch
2. build and run the app
3. run `npm run e2e:puppeteer datagrid`
4. It should pass.

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [ ] A note to the change log.

### CODE REVIEW

**Comment by ericangeles** [comment]
@janahintal can you also add a changelog? Thanks!

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -32,7 +32,8 @@
 
 ## v4.63.0 Features
 
-- `[File Upload]` Added close button on file error message ([#6229](https://github.com/infor-design/enterprise/issues/6229))
+- `[File Upload]` Added close button on file error message. ([#6229](https://github.com/infor-design/enterprise/issues/6229))
+- `[Datagrid]` Added puppeteer script for fallback image tooltip text. ([#6278](https://github.com/infor-design/enterprise/issues/6278))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:36]: Should be first in alpha order

**Review by tmcconechy — COMMENTED** [review]
Looks good but minor order issue in the log.

File: docs/CHANGELOG.md (hunk 1 above)
Comment (ericangeles) [reply docs/CHANGELOG.md:36]: @janahintal

**Review by ericangeles — APPROVED** [review]

**Pull request merged**